worker-workflow/src/test/resources/workflow-control-test.js

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
Is it really necessary to duplicate this code? I haven't dug into it but seems suspect to be using a copy of the real code in a test.